usr/share/rear/layout/save/default/445_guess_bootloader.sh

```diff
@@ -58,6 +58,13 @@ for block_device in /sys/block/* ; do
     Log "End of strings in the first bytes on $disk_device"
 done
 
+if [ "$ARCH" = "Linux-arm" ]; then
```

Comment:
@Lukey3332 
could you please move the
<pre>
if [ "$ARCH" = "Linux-arm" ]; then
    BOOTLOADER=ARM
    ...
    return
fi
</pre>
block as is just before the
<pre>
# Finally guess the used bootloader
...
</pre>
block
because a "finally guess" should rellay be the last step
cf. the example in the "Return early, return often" section in 
https://github.com/rear/rear/wiki/Coding-Style

Reply:
Do we need an arm-specific code block in the generic `default` directory? I would hope that all arm-specific code is in the `arm` directory.